client/cli_crypto_service.go

```diff
@@ -95,9 +120,8 @@ func (ccs *CryptoService) Sign(keyIDs []string, payload []byte) ([]data.Signatur
 	return signatures, nil
 }
 
-func sign(privKey *data.PrivateKey, hash crypto.Hash, hashed []byte) ([]byte, error) {
-	// TODO(diogo): Implement support for ECDSA.
-	if strings.ToLower(privKey.Cipher()) != "rsa" {
+func rsaSign(privKey *data.PrivateKey, hash crypto.Hash, hashed []byte) ([]byte, error) {
+	if strings.ToUpper(privKey.Cipher()) != "RSA" {
```

Comment:
I wonder if it would make sense to have `Cipher` do the `ToUpper` so we don't have the risk of forgetting to transform to uppercase for a string comparison.

I also think it would be a good idea to make the algorithms string constants we can refer to (something like `signer.AlgoRSA`) to guard against incorrect character case or typos, but this is more of a cosmetic thing. It would also help prevent confusion between cryptographic algorithms and signing methods.